addons/crm_iap_lead_enrich/data/message_template.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <template id="lead_enrich_message_not_found">
```

Comment:
I would rename to something like 'mail_message_lead_enrich_notfound'. Purpose is to name space: used as core of mail.message, linked to lead, then suffix to understand its purpose.